js/components/thermoscope.js

```diff
@@ -125,6 +125,10 @@ export default class Thermoscope extends PureComponent {
     let showClass = hidden ? "hidden" : "shown";
     let tempScale = paused ? zeroTempScale : model.tempScale;
 
+    // if mixing without live sensors, simulate a median temperature
+    let mixingTemperature = mixingValues && !liveData ? (Math.abs(mixingValues.aTemp - mixingValues.bTemp) / 2) + Math.min(mixingValues.aTemp, mixingValues.bTemp) : temperature;
+    let calculatedTemperature = mixingValues ? mixingTemperature : temperature;
```

Comment:
Is there a reason the median can't just be calculated by `(aTemp + bTemp) / 2`?

Median calculation aside, and using my other comment about pegged values, I might say something like this instead:

```
// if mixing without live sensors, simulate a median temperature
let calculatedTemperature = isFinite(aPegged) && isFinite(bPegged) && !liveData ? medianVal : temperature;
```

Reply:
Much better, thanks! Not sure what I was thinking!